pkgs/by-name/re/reposilite/package.nix

```diff
@@ -1,11 +1,20 @@
 {
-  stdenv,
   lib,
+  stdenv,
   fetchurl,
-  makeWrapper,
   jre_headless,
+  linkFarm,
+  makeWrapper,
+  plugins ? [ ],
 }:
-
+let
+  pluginsDir = linkFarm "reposilite-plugins" (
+    builtins.map (p: {
+      name = (builtins.parseDrvName p.name).name + ".jar";
```

Comment:
Can't we just use `"${p.name}.jar"`?

Reply:
Using `p.name` directly technically allow for multiple plugins of the same version, which I'm not sure Reposilite supports or people have reason to use. Using the equivalent of `pname` can help prevent any path conflicts in these cases and raise an error, so I'd say it has some purpose